### ISSUE TITLE
Compile error on a file which does not return CSS 

### ISSUE THREAD

**Issue by smumu** [brunch/sass-brunch#75]
Hello,

I have the same error as issue #68 : 
```
/usr/local/lib/node_modules/brunch/lib/fs_utils/pipeline.js:78
          throw new Error("Brunch SourceFile: file " + path + " data is invali
                ^
Error: Brunch SourceFile: file my_file.scss data is invalid
  at /usr/local/lib/node_modules/brunch/lib/fs_utils/pipeline.js:78:17
  at libsass.render.success (/home/samuel/www/braincracking/supv/www/node_modules/sass-brunch/index.js:99:7)
  at options.success (/home/samuel/www/braincracking/supv/www/node_modules/sass-brunch/node_modules/node-sass/lib/index.js:163:7)
```
But this issue is closed, so I open a new.

It seems that brunch expects a string as a compile result, but Sass send an object.
The file which cause that does not output CSS, it define a variable and a function or mixin, and in sass-brunch/index.js:97, we use `if (data.css) data = data.css` but data.css is an empty string.

I could fix it by changing with `if ('css' in data) data = data.css` but it will disappear when I'll do an update with NPM.

To conclude : if you want, I can make a pull request with this little fix, or maybe this performs is wanted. In this case, do you have a solution for me ?
Thank's in adance !

**Comment by es128** [comment]
Thanks for identifying the issue and a fix. Please do send a PR. Starting with a failing test would be preferable.

**Comment by ricardobanegas** [comment]
I still get the pipeline.js problem. Not getting this fix with the latest 1.8.10-version of sass-brunch. 

Steps to reproduce: 
* `npm install sass-brunch --save-dev`
* `sed -n 97,97p node_modules/sass-brunch/index.js`

Results in: `if(data.css) data = data.css;` should be `if('css' in data) data = data.css;` according to https://github.com/brunch/sass-brunch/commit/9ab5df8800b70b352fee29d0941cf8d5bced4f8e

**Comment by smumu** [comment]
According to the commits history (https://github.com/brunch/sass-brunch/commits/master), my fix is not in the 1.8.10, you'll need to wait for a new update ... In fact, there is no fix since mine and the 1.8.10 version.
While waiting for a developer answer or a new version, maybe you should apply my fix manually ?

**Comment by ricardobanegas** [comment]
Oh snap! Thanks for the info, yes already applied the patch but was hoping it was in the update so that I don't need to do an ugly fix for my team. 

I hope they release this fix soon so that I don't need to apply the fix manually, fix works great!

Kind regards
Ricardo

**Comment by ricardobanegas** [comment]
@paulmillr Have you planned a release of sass-brunch with @smumu fix? Thanks!

**Comment by es128** [comment]
just published 1.8.11 with the fix

**Comment by ricardobanegas** [comment]
Great thanks.
